src/abstractdataframe/io.jl

```diff
@@ -417,7 +423,12 @@ function printtable(io::IO,
             elseif isnothing(df[i, j])
                 print(io, nothingstring)
             else
-                if ! (etypes[j] <: Real)
+                if df[i,j] isa Markdown.MD
```

Comment:
Since accessing `df[i, j]` is relatively slow, better store it in `cell = df[i, j]` and use that afterwards as in the HTML method (not sure why we didn't do this here).

Reply:
Done.

Reply:
I don't see it, nor the other changes you mention below. :-)